uk.co.compucorp.civicrm.hrleaveandabsences/js/src/leave-absences/leave-type-wizard/components/leave-type-wizard.component.js

```diff
@@ -279,9 +324,18 @@ define([
      */
     function openNextTab () {
```

Comment:
there's something not quite right with this function, it does too many things: it either goes to the next tab or it goes to a different section altogether. Why this fork in the logic at all? In the view you clearly know whether a tab is the first or the last, so why do you need to delegate the switch to a different section to this function?

There are two options possible to fix this, from what i can see:
1) Either you rename this function to be more generic, so that it implies that it goes to the next "thing", whether it's a tab or a section
2) You directly call either "openNextTab" or "openNextSection" in the view, using the `ng-if` directive

Reply:
Done.